### PR TITLE
Click user details to navigate to my-site

### LINKED ISSUE #108: Navigate to my.realdevsquad.com
Upon clicking the user name and the profile pic, we should be redirected to my.realdevsquad.com

image

### PR DESCRIPTION
**Pull request by Vividh25**
Closes #108

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -70,9 +70,15 @@
...
         <div class="user-greet">
-          <div class="user-greet-msg">Hello text</div>
-          <img class="user-profile-pic" />
+          <div class="user-greet-hello">Hello,</div>
+          <a href="https://my.realdevsquad.com/">
+            <div class="user-greet-msg">username</div>
+          </a>
+          <a href="https://my.realdevsquad.com/">
+            <img class="user-profile-pic" />
+          </a>
```
Comment (ankushdharkar) [inline index.html:81]: Can we combine this to just one reference?

**Review by ankushdharkar — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Vividh25) [reply index.html:81]: Sure. On it

File: js/main.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ function setUserGreeting(username, firstName) {
     const msgGreetMsgEl = document.querySelector(".user-greet-msg");
     const userImgEl = document.querySelector(".user-profile-pic");
 
-    const greetMsg = `Hello, ${firstName}!`;
+    const greetMsg = `${firstName}!`;
```
Comment (swarajpure) [inline js/main.js:9]: This will now cause the output to look like `Hello,Vividh` instead of `Hello, Vividh`
(talking about the space after the comma)

Comment (Vividh25) [reply js/main.js:9]: Ohh I see, I'll fix it asap

Comment (Vividh25) [reply js/main.js:9]: This is how it look like right now.. is it fine ? 

hellotxt

Comment (swarajpure) [reply js/main.js:9]: Yes 🙂

**Pull request closed without merging**